### PR TITLE
Remove FLOAT SQL datatype

### PR DESCRIPTION
**Pull request by mihaibudiu**
Is this a user-visible change (yes/no): yes

### CODE REVIEW

**Comment by mihaibudiu** [comment]
Fixes #950

**Comment by gz** [comment]
This needs some code removal in the web-console too @Karakatiza666

File: CHANGELOG.md [hunk 1]
```diff
@@ -10,6 +10,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [Unreleased]
 
 ### Added
+
+- Removed suppoer for FLOAT SQL data type, since it is ambiguous.
```
Comment (gz) [inline CHANGELOG.md:14]:
```suggestion
### Removed

- Removed support for FLOAT SQL data type, since it is ambiguous.

### Added
```

**Review by gz — APPROVED** [review]

File: docs/sql/types.md [hunk 2]
```diff
@@ -17,7 +17,7 @@ The compiler supports the following SQL data types:
   digits) and a scale (number of decimal digits after period).  For example,
   23.456 has a precision of 5 and a scale of 3.  If scale is missing it is
   assumed to be 0.
-- `FLOAT`, or `FLOAT4`, or `FLOAT32`, or `REAL`, an IEEE 32-bit floating point number
+- `REAL`, or `FLOAT4`, or `FLOAT32`, an IEEE 32-bit floating point number
```
Comment (lalithsuresh) [inline docs/sql/types.md:20]: Are FLOAT4 and FLOAT32 accepted types?

**Review by lalithsuresh — APPROVED** [review]

File: docs/sql/types.md (hunk 2 above)
Comment (mihaibudiu) [reply docs/sql/types.md:20]: yes

**Pull request merged**